src/amp-story-player/amp-story-player-impl.js

```diff
@@ -144,6 +144,9 @@ export class AmpStoryPlayer {
   attachCallbacksToElement_() {
     this.element_.load = this.load.bind(this);
     this.element_.show = this.show.bind(this);
+    this.element_.play = this.play.bind(this);
+    this.element_.pause = this.pause.bind(this);
+    this.element_.togglePaused = this.togglePaused.bind(this);
```

Comment:
Let's keep the API simple and intuitive and just have `play()` and `pause()` for now.

Reply:
Done in latest commit